### ISSUE TITLE
No maximize botton on Ubuntu budgie 20.04

### ISSUE THREAD

**Issue by kubwit** [darktable-org/darktable#5273]
Hi.
Have Darktable 3 installed on ubuntu budgie and i have problems with screensizing (also on kubuntu) on mu laptop 15". But in Budgie desktop there is no way make it fit on screen. There is the maximize botton completely missing.
I tried with gnome tweak but all is how it should be. also all other apps have the maximize botton.

Any idea to this. Somebod ycan help?

**Comment by ptilopteri** [comment]
isn't that button a window manager function rather than a dt function?

**Comment by Nilvus** [comment]
You probably use a resolution lower than 1920x1080. I suppose you use last darktable 3, so 3.0.2. darktable set a minimum width size to let minimal space on the image. So just reduce panels (they are resizable dynamically with mouse). Then you will be able to resize window width and see the maximize button come back. It's a normal behavior.

**Comment by kubwit** [comment]
> isn't that button a window manager function rather than a dt function?

Thanks for quick reply. Yes i tohought the same that's why i looked first in WM to solve



> You probably use a resolution lower than 1920x1080. I suppose you use last darktable 3, so 3.0.2. darktable set a minimum width size to let minimal space on the image. So just reduce panels (they are resizable dynamically with mouse). Then you will be able to resize window width and see the maximize button come back. It's a normal behavior.

Thanks for your reply. Yes the max resolution i have is 1366x768 and yes it is 3.0.2. I try to reduce the panels but the max width of the whole app is larger than my screen.... And i cant resize it as much to make it fit.
Strange that on Kubuntu 20.04 same PC i don't have this issue
 Some idea?

**Comment by Nilvus** [comment]
> Thanks for your reply. Yes the max resolution i have is 1366x768 and yes it is 3.0.2. I try to reduce the panels but the max width of the whole app is larger than my screen.... And i cant resize it as much to make it fit.
> Strange that on Kubuntu 20.04 same PC i don't have this issue
> Some idea?

It's a too small resolution for editing photos, so for a software like darktable (I don't say it's not possible just that it's small). Sorry, but as I don't know anything else about your system, I can't help but there's for sure a difference like:

- screen resolution of course but you probably use same
- font set (family and size) on your desktop environment (a condensed one or lower size would help to reduce more the window width.
- DPI factor (check that screen_dpi_overwrite is set to -1.0 in your darktablerc file
- different scale factor used
- some inner system on one of your desktop environment to render Gtk apps (for example, it's only that on Kde 5.18 that Gtk apps are really rendered correctly)

Anyway, the best way would be to use a more recommended resolution, so a higher one.

**Comment by kubwit** [comment]
> > Thanks for your reply. Yes the max resolution i have is 1366x768 and yes it is 3.0.2. I try to reduce the panels but the max width of the whole app is larger than my screen.... And i cant resize it as much to make it fit.
> > Strange that on Kubuntu 20.04 same PC i don't have this issue
> > Some idea?
> 
> It's a too small resolution for editing photos, so for a software like darktable (I don't say it's not possible just that it's small). Sorry, but as I don't know anything else about your system, I can't help but there's for sure a difference like:
> 
>     * screen resolution of course but you probably use same
> 
>     * font set (family and size) on your desktop environment (a condensed one or lower size would help to reduce more the window width.
> 
>     * DPI factor (check that screen_dpi_overwrite is set to -1.0 in your darktablerc file
> 
>     * different scale factor used
> 
>     * some inner system on one of your desktop environment to render Gtk apps (for example, it's only that on Kde 5.18 that Gtk apps are really rendered correctly)
> 
> 
> Anyway, the best way would be to use a more recommended resolution, so a higher one.

Thanks a lot for your time to reply. Yes it is not a problem. i work around or use kde.. I just wondered. If i can be for any help (providing info or testing) if there is an interest i am happy to provide these info. Anyway my laptop is lenovo g50-70 and on budgie i can't get higher resolution. donnow why.
00:02.0 VGA compatible controller: Intel Corporation Haswell-ULT Integrated Graphics Controller (rev 0b) (prog-if 00 [VGA controller])
	Subsystem: Lenovo Haswell-ULT Integrated Graphics Controller
	Flags: bus master, fast devsel, latency 0, IRQ 47
	Memory at d0000000 (64-bit, non-prefetchable) [size=4M]
	Memory at c0000000 (64-bit, prefetchable) [size=256M]
	I/O ports at 6000 [size=64]
	Expansion ROM at 000c0000 [virtual] [disabled] [size=128K]
	Capabilities: [90] MSI: Enable+ Count=1/1 Maskable- 64bit-
	Capabilities: [d0] Power Management version 2
	Capabilities: [a4] PCI Advanced Features
	Kernel driver in use: i915
	Kernel modules: i915
Thanks

**Comment by Nilvus** [comment]
I wasn't asking for your hardware configuration but about settings I cited. Hardware configuration is not useful here. I don't see anything else that I pointed. So think about closing your issue now. Thanks.

**Comment by kubwit** [comment]
Ok. Just to give you a positve feedback. It was the resolution problem you mentioned. I got maximum resolution as above mentioned and i tried to figure out how to get a better resolution as it wasn't possible to select in my both ubuntu 20.04 installations (budgie and kubuntu).

I followed this https://www.youtube.com/watch?v=LiP-YqtZoNQ and now i'm running 1920x1080.

And yes it works

Thanks for your replies. I close
